sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,16 @@
+-- Baby Blizzard Bear
+UPDATE `creature_template` SET `AIName`='SmartAI',`ScriptName`='' WHERE `entry`=32841;
+REPLACE INTO `creature_template_addon` (`entry`, `auras`) VALUES (32841,'61854');
+DELETE FROM `smart_scripts` WHERE `entryorguid`=32841 AND `source_type`=0;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=3284100 AND `source_type`=9;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(32841,0,0,0,8,0,100,0,61853,0,0,0,80,3284100,0,0,0,0,0,1,0,0,0,0,0,0,0,'Baby Blizzard Bear - On Spell Hit - Action list'),
```

Comment:
Add a repeat delay superior to the action list, 15s or more, even if the spell is triggered with Interval: 30 seconds